MapboxNavigation/Route.swift

```diff
@@ -20,7 +20,7 @@ extension Route {
         let precedingStepCoordinates = precedingSteps.compactMap { $0.coordinates }.reduce([], +)
         let precedingPolyline = Polyline((precedingLegCoordinates + precedingStepCoordinates).reversed())
         
-        let followingLegs = legs.prefix(upTo: legIndex)
+        let followingLegs = legs.suffix(from: legIndex).dropFirst()
```

Comment:
Thanks for catching this scrumptious bit of copy-pasta. 🍝 By the way, it should also be safe to say `legs.suffix(from: legIndex + 1)`, but this is fine.